STEP_Project/tests/popupTest.js

```diff
@@ -36,26 +36,33 @@ describe('Testing pageType that determines enabling/disabling of buttons dependi
 /**
  * testing function disableButtons
  */
+function initTest()
+{
+  const arButton = document.createElement('BUTTON')
+  arButton.setAttribute("id", "addRemove");
+  const sButton = document.createElement('BUTTON')
+  sButton.setAttribute("id", "select");
+  document.body.appendChild(arButton);
+  document.body.appendChild(sButton);
+  s = document.getElementById("select");
```

Comment:
Rename this to something like selectButton and similar for "ar" variable.

Reply:
variables renamed